seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2636,7 +2636,7 @@
         "<blockquote>function plusThree(num) {<br>  return num + 3;<br>}<br>var answer = plusThree(5); // 8</blockquote>",
         "<code>plusThree</code> takes an <dfn>argument</dfn> for <code>num</code> and returns a value equal to <code>num + 3</code>.",
         "<hr>",
-        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value."
+        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value. You can call the <code>timesFive</code> function to test it."
```

Comment:
@systimotic no worries! There's a lot of activity on this repo to keep track of 😄 

That seed will also cause problems (at least as it is right now) because `timesFive` hasn't been defined yet and will cause errors, like I've shown in my comment https://github.com/freeCodeCamp/freeCodeCamp/pull/9651/files#r71084020

@Greenheart true, might as well keep it in.